plugins/amdgpu/README.md

```diff
@@ -0,0 +1,275 @@
+Supporting ROCm with CRIU
+=========================
```

Comment:
This file should be added with commit "criu/plugin: Support AMD ROCm Checkpoint Restore with KFD".

Reply:
ack